pages/rdm_cycle/planning.md

```diff
@@ -5,14 +5,37 @@ contributors:
 ---
```

Comment:
Please, make sure to add the names of the contributors

Reply:
I added them, is it enough that I saved the draft or do I need to do a pull request?